charts/rancher-cis-benchmark/0.2.0/Chart.yaml

```diff
@@ -10,3 +10,4 @@ sources:
 maintainers:
   - name: Murali Paluru
     email: leodotcloud@gmail.com
+kubeVersion: '< 1.22.0-0'
```

Comment:
Adding a `kubeVersion` to an already released chart means that any person who installed CIS v1 onto a k8s 1.22 cluster **before** this change is merged will end up being unable to do an in-place upgrade on it.

@jiaqiluo is it possible that someone could be using CIS v1 in an imported 1.22 cluster today? If so, I think this change needs to be reverted, unless we are okay with the risk here.